src/validation/fields/helper/github-handles.validation.helper.ts

```diff
@@ -11,7 +11,7 @@ export const validateGithubHandles = (values: string) => {
 
     for (const handle of handles) {
         if (!githubHandleRegex.test(handle) || handle.length > githubHandlesMaxLength || handle.length < 1) {
```

Comment:
You can remove `githubHandlesMaxLength` variable and `handle.length > githubHandlesMaxLength` as the regex covers this check.